packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/uploadView.ts

```diff
@@ -79,6 +80,19 @@ export const DEFAULT_UPLOAD_VIEW_DISPLAY_TEXT: DefaultUploadViewDisplayText = {
 
     return { content: 'All files uploaded.', type };
   },
+  getFilesValidationMessage: ({ invalidFiles } = {}) => {
+    if (!invalidFiles) {
+      return undefined;
+    }
+    const fileNames = invalidFiles
```

Comment:
Sanity check: invalidFiles can be an empty array here which could generate an incomplete error message, will this be expected?

Reply:
It cannot be empty array according to how we populate the array, but I will handle the scenario here as well.